tests/integration.rs

```diff
@@ -5610,7 +5673,10 @@ mod parallel {
         #[cfg(target_arch = "x86_64")]
         let focal_image = FOCAL_IMAGE_NAME.to_string();
         let focal = UbuntuDiskConfig::new(focal_image);
-        let guest = Guest::new(Box::new(focal));
+        let guest = Guest::new(
+            Box::new(focal),
+            Some(fw_path(FwType::RustHypervisorFirmware)),
```

Comment:
This is the only place where I found to be odd by following the original code path, that is by default we always invoke `fw_path(FwType::RustHypervisorFirmware)` when creating a new `Guest` even for `aarch64` (that uses its own hard-coded firmware name and ignores the provided `FwType`). 

If we are fine with changing the original code path, we can adjust to use `fn fw_path(#[cfg(target_arch = "x86_64")] fw_type: FwType) -> String` and just invoke `fw_path()` for `aarch64`. Any preference? @sboeuf

Reply:
Following my suggestion about removing `fw_path` from `Guest`, I think that should simplify everything here since you'd simply invoke the local `fw_path()` for both architectures.